resources/js/components/felte-form/FelteForm.tsx

```diff
@@ -1,20 +1,14 @@
-import {
-  Form,
-  FormConfigWithoutTransformFn,
-  KnownHelpers,
-  Obj,
-  Paths,
-} from "@felte/core";
-import { reporter } from "@felte/reporter-solid";
-import { createForm } from "@felte/solid";
-import { type KnownStores } from "@felte/solid/dist/esm/create-accessor";
-import { validator } from "@felte/validator-zod";
-import { useTransContext } from "@mbarzda/solid-i18next";
-import { isAxiosError } from "axios";
-import { Api } from "data-access/memo-api/types";
-import { Context, JSX, createContext, splitProps, useContext } from "solid-js";
+import {Form, FormConfigWithoutTransformFn, KnownHelpers, Obj, Paths} from "@felte/core";
+import {reporter} from "@felte/reporter-solid";
+import {createForm} from "@felte/solid";
```

Comment:
jestem na tyle przyzwyczajony do spacji w { } że aż dziwnie mi się to czyta :D

Reply:
A mi dziwnie ze spacjami :) jeszcze importy to pół biedy ale obiekty to bardzo dziwnie. Takie życie :)